astropy/time/core.py

```diff
@@ -1529,9 +1531,9 @@ def _advanced_index(self, indices, axis=None, keepdims=False):
         if keepdims and indices.ndim < self.ndim:
             indices = np.expand_dims(indices, axis)
         return tuple([(indices if i == axis else np.arange(s).reshape(
-            (1,)*(i if keepdims or i < axis else i-1) + (s,) +
-            (1,)*(ndim-i-(1 if keepdims or i > axis else 2))))
-                for i, s in enumerate(self.shape)])
+            (1,) * (i if keepdims or i < axis else i - 1) + (s,)
+            + (1,) * (ndim - i - (1 if keepdims or i > axis else 2))))
```

Comment:
Here, I partially like the extra spaces (between tuple and `*` and multiplier), but less so for the numbers inside the multipliers; same link as above, but more directly applicable: https://www.python.org/dev/peps/pep-0008/#other-recommendations